app/femr/util/startup/DatabaseSeeder.java

```diff
@@ -1042,6 +1044,149 @@ private void seedDefaultTabFieldSizes() {
+            }
+            if (!containMedicationAdministration(administrations, "qHS")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("qHS");
+                medicationAdministration.setDailyModifier(1f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
+            if (!containMedicationAdministration(administrations, "QID")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("QID");
+                medicationAdministration.setDailyModifier(4f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
+            if (!containMedicationAdministration(administrations, "q5min")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("q5min");
+                medicationAdministration.setDailyModifier(288f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
+            if (!containMedicationAdministration(administrations, "qOd")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("qOd");
+                medicationAdministration.setDailyModifier(0.5f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
+            if (!containMedicationAdministration(administrations, "qPM")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("qPM");
+                medicationAdministration.setDailyModifier(1f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
+            if (!containMedicationAdministration(administrations, "q week")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("q week");
+                medicationAdministration.setDailyModifier(0.142857f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
+            if (!containMedicationAdministration(administrations, "TID")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("TID");
+                medicationAdministration.setDailyModifier(3f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
+            if (!containMedicationAdministration(administrations, "TIW")) {
+                medicationAdministration = new MedicationAdministration();
+                medicationAdministration.setName("TIW");
+                medicationAdministration.setDailyModifier(0.42857f);
+                medicationAdministrationsToAdd.add(medicationAdministration);
+            }
```

Comment:
The daily modifier values listed in your screenshot and those in the database seeder do not match up. Am I missing something?